### ISSUE TITLE
Test mempool and message size when submitting lengthy parameter values

### ISSUE THREAD

**Issue by shannonwells** [LibertyDSNP/frequency#97]
Issues:
* In storage we are storing a BoundedVec and right now that bound (SchemaMaxBytesBoundedVecLimit) is set in a Config and also used by the `schema` parameter in `register_schema`
* Additionally there is a `GovernanceSchemaMaxBytes` in storage which can be set and is used in the extrinsic.

Questions to answer:
1. Do we really need both values?
    - Yes 
1. What happens when a parameter for an extrinsic is a boundedvec and you attempt to register a schema that is larger than `SchemaMaxBytesBoundedVecLimit`?
    1. Does this message get gossiped?
    2. Does it panic?
    3. Is it truncated? 
1. What's the effect on the node/network of attempting to register a lot of very long schemas?

**Comment by enddynayn** [comment]
We can ask Parity if this is a concern we should take into account.

**Comment by dopatraman** [comment]
Results of the investigation:

**Methods:**
I tried to register schemas of various lengths: 65_000, 493_121, 1_000_000, and all were barred from entering the transaction queue. I ran batches of 1000 messages each, and all were prevented from entering the network with no apparent adverse effects on the network itself.


**1. Does this message get gossiped?**
It appears as if if it does **not** get gossiped. Messages do not seem to enter the transaction queue.

**2. Does it panic?**
Yes. The error that gets posted is this:
```
panicked at 'Bad input data provided to validate_transaction: Codec error',/frequency/runtime/frequency-rococo/src/lib.rs:799:1    
```

**3. Is it truncated?**
It seems as if the message is barred outright. I did not observe any truncation.
